### ISSUE TITLE
A question about "Few-Shot".

### ISSUE THREAD

**Issue by QWTforGithub** [Pang-Yatian/Point-MAE#14]
Thank you very much for your help last time.  I have a question about "fee-shot ". In the training set," 5way-10shot " means that we randomly selects 5 classes and has 10 samples for each class.  May I ask how to select classes and samples in the test set?  Is it still the same 5 classes with10 samples ?  Or all of the samples for each class?

**Comment by Pang-Yatian** [comment]
Hi,

You may refer to our paper, in which the setting of few-shot learning is described in detail. 

The short answer is that we select 20 UNSEEN samples for each of the 5 classes.

**Comment by QWTforGithub** [comment]
> Hi,
> 
> You may refer to our paper, in which the setting of few-shot learning is described in detail.
> 
> The short answer is that we select 20 UNSEEN samples for each of the 5 classes.

Sorry, I found the answer in your article shortly after posting the question.

**Comment by Pang-Yatian** [comment]
Never mind.

**Comment by Pang-Yatian** [comment]
> I have another question: in the paper you report mean accuracies with std from 10 independent experiments. Are those w.r.t. the final epoch or best epoch of each of the 10 experiments?

Hi,

We report the best epoch results.
